docs/case-studies/jenkins-cicd.md

```diff
@@ -2,19 +2,19 @@
 title:  Jenkins + KubeVela for CI/CD
 ---
 
-This section will introduce how to combine KubeVela and Jenkins to do CI (continuous integration) and CD (continuous delivery). The following example is based on the publish process of a simple HTTP server. The code can be found at [here](https://github.com/Somefive/KubeVela-demo-CICD-app).
+This section will introduce how to combine KubeVela with Jenkins to do CI (continuous integration) and CD (continuous delivery). The following example is based on the release process of a simple HTTP server. The code can be found at [here](https://github.com/Somefive/KubeVela-demo-CICD-app).
 
 ## Preparation
 
 Before combining KubeVela and Jenkins, you need to ensure the following environments have already been set up.
 
 1. Deploy Jenkins service with Docker support, including related plugins and credentials which will be used to access image repository.
```

Comment:
How can make sure this step is ready?

Reply:
The case study here assumes user has experiences in Jenkins + GitHub Webhook and has already set up them. For users not sure how to set up these matters, there is the blog for introducing it step by step.